src/backend/gl/src/state.rs

```diff
@@ -20,11 +20,11 @@ pub fn bind_polygon_mode(gl: &gl::Gl, mode: pso::PolygonMode, bias: Option<pso::
     unsafe { gl.PolygonMode(gl::FRONT_AND_BACK, gl_draw) };
 
     match bias {
-        Some(bias) => unsafe {
+        Some(pso::State::Static(bias)) => unsafe {
```

Comment:
Isn't it possible for a dynamic depth bias to have been set at this point?

Otherwise, where would the dynamic depth bias be set?

Reply:
Yeah, the problem is that we haven't had a dynamic depth bias exposed at all, and I want to limit the scope of this PR to providing this in API and implementing in Metal and Vulkan only. Other backends are not regressed, they will just need to catch up.